### PR TITLE
Zip

### PR DESCRIPTION
**Pull request by brarcher**
Adds support  for import/exporting to/from Zip files.

### CODE REVIEW

**Comment by brarcher** [comment]
@paulle I'm adding a feature to Budget Watch which will allow one to export the application's data in Zip format, which will include receipt images. For this, the following string is being added:

<string name="fileFormat">File format</string>

Would you mind providing a translation into German for this string?

Thanks!

**Comment by paulle** [comment]
please send me the to translate. Can't find it.


Am 06.09.2016 um 16:57 schrieb Branden Archer:

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -76,8 +76,9 @@
     <string name="importExport">Import/Export</string>
     <string name="importName">Import</string>
     <string name="exportName">Export</string>
-    <string name="importExportHelp">Data is imported to/exported from BudgetWatch.csv on external storage</string>
+    <string name="importExportHelp">Data is imported to/exported from BudgetWatch.<xliff:g id="extension">%s</xliff:g> on external storage</string>
     <string name="importedFrom">Imported from: %1$s</string>
+    <string name="fileFormat">File format</string>
```
Comment (brarcher) [inline app/src/main/res/values/strings.xml]: @paulle  This is the only string that needs to be translated, namely "fileFormat". If it is easy, you can simply respond in a comment with the translation and I'll handle putting it in the correct place.

Comment (paulle) [inline app/src/main/res/values/strings.xml]: <string name="importedFrom">Imported from: %1$s</string>
<string name="fileFormat">File format</string>


<string name="importiert von">Imported from: %1$s</string>
<string name="Dateiformat">File format</string>

here you have it, not too much to translate

Am 07.09.2016 um 01:20 schrieb Branden Archer:
> <string name="importedFrom">Imported from: %1$s</string>
> > +    <string name="fileFormat">File format</string>

**Comment by brarcher** [comment]
@paulle I think you meant:

<string name="fileFormat">Datei format</string>

Thanks!

**Pull request merged**